docs/utilities/batch.md

```diff
@@ -0,0 +1,258 @@
+---
+title: SQS Batch Processing
+description: Utility
+---
+
+The SQS batch processing utility provides a way to handle partial failures when processing batches of messages from SQS.
+
+**Key Features**
+
+* Prevent successfully processed messages from being returned to SQS
+* A simple interface for individually processing messages from a batch
+
+**Background**
+
+When using SQS as a Lambda event source mapping, Lambda functions can be triggered with a batch of messages from SQS.
+
+If your function fails to process any message from the batch, the entire batch returns to your SQS queue, and your Lambda function will be triggered with the same batch again.
+
+With this utility, messages within a batch will be handled individually - only messages that were not successfully processed
+are returned to the queue.
+
+!!! warning
+    While this utility lowers the chance of processing messages more than once, it is not guaranteed. We recommend implementing processing logic in an idempotent manner wherever possible.
+    More details on how Lambda works with SQS can be found in the [AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html)
+
+## Install
+
+To install this utility, add the following dependency to your project.
+
+=== "Maven"
+    ```xml
+    <dependency>
+        <groupId>software.amazon.lambda</groupId>
+        <artifactId>powertools-sqs</artifactId>
+        <version>1.2.0</version>
+    </dependency>
+    ```
+=== "Maven Configuration"
+
+    Configure the aspectj-maven-plugin to compile-time weave (CTW) the
+    aws-lambda-powertools-java aspects into your project. You may already have this
+    plugin in your pom. In that case add the dependency to the `aspectLibraries`
+    section.
+
+    ```xml hl_lines="13 14 15 16"
+    <build>
+        <plugins>
+            ...
+            <plugin>
+                 <groupId>org.codehaus.mojo</groupId>
+                 <artifactId>aspectj-maven-plugin</artifactId>
+                 <version>1.11</version>
+                 <configuration>
+                     <source>1.8</source>
+                     <target>1.8</target>
+                     <complianceLevel>1.8</complianceLevel>
+                     <aspectLibraries>
+                         <aspectLibrary>
+                             <groupId>software.amazon.lambda</groupId>
+                             <artifactId>powertools-sqs</artifactId>
+                         </aspectLibrary>
+                     </aspectLibraries>
+                 </configuration>
+                 <executions>
+                     <execution>
+                         <goals>
+                             <goal>compile</goal>
+                         </goals>
+                     </execution>
+                 </executions>
+            </plugin>
+            ...
+        </plugins>
+    </build>
+    ```
```

Comment:
Similar to "Homepage" page, this can be in one tab, since we need the content from both tabs to configure the pom.